foxtrot-server/src/main/java/com/flipkart/foxtrot/server/utils/FoxtrotLoggerUtils.java

```diff
@@ -0,0 +1,72 @@
+package com.flipkart.foxtrot.server.utils;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.filter.ThresholdFilter;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.FileAppender;
+import ch.qos.logback.core.rolling.FixedWindowRollingPolicy;
+import ch.qos.logback.core.rolling.RollingFileAppender;
+import ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy;
+import ch.qos.logback.core.spi.FilterAttachable;
+import com.flipkart.foxtrot.server.config.LogRotationFileConfig;
+import com.google.common.base.Optional;
+import com.yammer.dropwizard.config.LoggingConfiguration;
+import com.yammer.dropwizard.logging.LogFormatter;
+
+/**
+ * Created by anzer.moid on 10/02/18.
+ */
+public class FoxtrotLoggerUtils {
+    public static FileAppender<ILoggingEvent> buildFileAppender(LoggingConfiguration.FileConfiguration file,
```

Comment:
@anzrmoid  
Is there any particular reason why we're creating the appenders explicitly ?
Is appender config from .yml not working here ?

Reply:
Agreed. This can be done from logger config.

Reply:
Foxtrot is on older version of dropwizard 0.6.2 which doesn't support size based configs. hence, explicit creation of appenders.